controllers/webspherelibertyapplication_controller.go

```diff
@@ -476,8 +476,8 @@ func (r *ReconcileWebSphereLiberty) Reconcile(ctx context.Context, request ctrl.
 		err = r.CreateOrUpdate(deploy, instance, func() error {
 			oputils.CustomizeDeployment(deploy, instance)
 			oputils.CustomizePodSpec(&deploy.Spec.Template, instance)
-			if err := lutils.CustomizeLibertyEnv(&deploy.Spec.Template, instance, r.GetClient()); err != nil {
-				reqLogger.Error(err, "Failed to reconcile Liberty env")
+			if err := lutils.CustomizeLibertyEnv(&deploy.Spec.Template, instance, r.GetClient(), ba); err != nil {
```

Comment:
`instance` parameter, that's already passed, is an 'implementation' of BaseComponent, so that can be used to get manageTLS